modules/mobile/app/services/mobile/v0/appointments/proxy.rb

```diff
@@ -131,7 +139,21 @@ def fetch_appointments(start_date, end_date)
           { va: va_response, cc: cc_response, requests: requests_response }
         end
 
-        def fetch_facilities(appointments)
+        def new_fetch_facilities(appointments)
+          facility_ids = appointments.map(&:id_for_address).uniq
+
+          ids = ids(facility_ids) # may not be needed
```

Comment:
Since ids are formatted in Vets API facility controller as a csv first before using MFS's get_facilities, I had to check if it had to be set up this way as well within mobile. I forgot to remove the comment..